### PR TITLE
PN-13041: additional languages in schema

### PR DESCRIPTION
**Pull request by fbianchicodermine**
## Short description

Add additional languages in new notification schema

## List of changes proposed in this pull request

- Update version of `pn-delivery` service in `pom`

### CODE REVIEW

File: docs/openapi/notification-schemas/notification.yaml [hunk 1]
```diff
@@ -243,6 +243,11 @@ components:
           pattern: "^([0-9]{6}[A-Z]{1})$"
           description: >-
             Codice tassonomico della notifica basato sulla definizione presente nell'allegato 2 capitolo C del bando [__AVVISO PUBBLICO MISURA 1.4.5 PIATTAFORMA NOTIFICHE DIGITALI__](https://pnrrcomuni.fondazioneifel.it/bandi_public/Bando/325)
+        additionalLanguages:
+          type: array
+          items:
+            type: string
+            maxItems: 1
```
Comment (alessandrogelmi) [inline docs/openapi/notification-schemas/notification.yaml:250]: Add a description

File: src/main/java/it/pagopa/pn/bff/mappers/notifications/NewSentNotificationMapper.java [hunk 2]
```diff
@@ -22,7 +22,7 @@ public interface NewSentNotificationMapper {
      * @param request the BffNewNotificationRequest to map
      * @return the mapped NewNotificationRequestV23
```
Comment (alessandrogelmi) [inline src/main/java/it/pagopa/pn/bff/mappers/notifications/NewSentNotificationMapper.java:23]: Update java doc

File: src/main/java/it/pagopa/pn/bff/mappers/notifications/NotificationSentDetailMapper.java [hunk 3]
```diff
@@ -25,7 +25,7 @@ public interface NotificationSentDetailMapper {
      * @param notification the FullSentNotificationV24 to map
```
Comment (alessandrogelmi) [inline src/main/java/it/pagopa/pn/bff/mappers/notifications/NotificationSentDetailMapper.java:25]: Update java doc

**Review by alessandrogelmi — CHANGES_REQUESTED** [review]

**Review by alessandrogelmi — APPROVED** [review]

**Review by alessandrogelmi — CHANGES_REQUESTED** [review]
Merge develop into this PR in order to take the updated commit id of pn-delivery

**Review by AndreaCimini90 — APPROVED** [review]

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**